flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinBlockFlinkInputFormat.java

```diff
@@ -32,52 +32,48 @@
 
 
 public class BitcoinBlockFlinkInputFormat extends AbstractBitcoinFlinkInputFormat<BitcoinBlock> implements CheckpointableInputFormat<FileInputSplit, Long> {
-	
+
 
 
 	private static final Log LOG = LogFactory.getLog(BitcoinBlockFlinkInputFormat.class.getName());
 	/**
-	 * 
+	 *
 	 */
 	private static final long serialVersionUID = 4150883073922261077L;
 	private boolean isEndReached;
-	
+
 	public BitcoinBlockFlinkInputFormat(int maxSizeBitcoinBlock,  String specificMagicStr,
-			boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
+										boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
 		this(maxSizeBitcoinBlock, specificMagicStr, useDirectBuffer,false);
 	}
-	
+
 	public BitcoinBlockFlinkInputFormat(int maxSizeBitcoinBlock,  String specificMagicStr,
-			boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
+										boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
 		super(maxSizeBitcoinBlock, specificMagicStr, useDirectBuffer);
 		this.isEndReached=false;
 	}
-	
-	@Override
```

Comment:
I do not think we can remove it. The super class AbstractBitcoinFlinkInputFormat has now additionally the same variable, but in the end the variable isEndReached in AbstractBitcoinFlinkInputFormat is completely different from isEndReached in BitcoinBlockFlinkInputFormat

Reply:
it is also a bit confusing now

Reply:
Acknowledged. The idea was to Pull the Method "**isEndReached()**" up to the abstract class to remove code duplication across the following classes:
- flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinBlockFlinkInputFormat.java
- flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinRawBlockFlinkInputFormat.java
- flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinTransactionFlinkInputFormat.java